test/test_tensor_creation_ops.py

```diff
@@ -739,8 +745,7 @@ def test_cat_out_memory_format(self, device):
         self.assertTrue(res1_cpu.is_contiguous(memory_format=torch.contiguous_format))
 
         # Case 2: if out= is not the correct shape then the output it is resized internally
-        # - For the CPU variant the memory format is that of the first tensor
-        # - For the CUDA variant it only propagates memory format if all the tensors have
+        # - For both CPU and CUDA variants, it only propagates memory format if all the tensors have
```

Comment:
@ngimel, is this change in memory format propagation on CPU an issue? It seems useful to clean this logic up so it's backend-agnostic, but it sounds mildly BC-breaking.